stl/debugger/STL.natvis

```diff
@@ -1955,4 +2043,100 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
     <Expand/>
   </Type>
 
+  <Type Name="std::formatter&lt;*,*&gt;">
+    <DisplayString>{_Specs._Dynamic_width_index}, {_Specs._Dynamic_precision_index}</DisplayString>
+    <Expand>
+      <Item Name="Width index">_Specs._Dynamic_width_index</Item>
+      <Item Name="Precision index">_Specs._Dynamic_precision_index</Item>
+    </Expand>
+  </Type>
+
+  <Type Name="std::basic_format_parse_context&lt;*&gt;">
+    <DisplayString>{_Format_string}</DisplayString>
+    <Expand>
+      <Item Name="Arguments">_Num_args</Item>
+      <Item Name="Next arg id">_Next_arg_id</Item>
+    </Expand>
+  </Type>
+
+  <Type Name="std::basic_format_args&lt;*&gt;">
+    <DisplayString>Num args:{_Num_args}</DisplayString>
+    <Expand>
+      <Item Name="Index">_Index_array->_Index</Item>
+      <Item Name="Type">_Index_array->_Type_</Item>

```

Comment:
can this show the storage that's "off the end" I'm not sure it's useful to visualize the format of basic_format_args if the debugger can't actually parse the arg storage